packages/shell-api/src/shell-instance-state.ts

```diff
@@ -217,6 +220,8 @@ export default class ShellInstanceState {
   }
 
   async fetchConnectionInfo(): Promise<ConnectionInfo | undefined> {
+    const { version } = require('../package.json');
```

Comment:
This will currently work fine because we're releasing packages using the same version, but there's no guarantee that that would stay that way indefinitely – are you sure this is what we need?